storybook-testing/teardown.ts

```diff
@@ -0,0 +1,5 @@
+const exec = require('child_process').execSync;
+async function makeReport() {
```

Comment:
we now have like 3 teardown files for different tests, maybe some portions of the teardown scripts could be shared?